draft-ietf-tls-rfc4492bis.xml

```diff
@@ -339,7 +340,10 @@
             deprecates all but the uncompressed point format. Implementations of this document MUST support the
             uncompressed format for all of their supported curves, and MUST NOT support other formats for curves
             defined in this specification. For backwards compatibility purposes, the point format list extension
-            MUST still be included, and contain exactly one value: the uncompressed point format (0).</t>
+            MAY still be included, and contain exactly one value: the uncompressed point format (0). RFC 4492 
+            specified that if this extension is missing, it means that only the uncompressed point format is 
+            supported, so interoperability with implementations that support the uncompressed format should work
+            with or without the extension.</t>
           <t> The ECPointFormat name space is maintained by IANA.  See <xref target="iana"/> for information on how
             new value assignments are added.</t>
           <t> Items in ec_point_format_list are ordered according to the client's preferences (favorite choice first).</t>
```

Comment:
I think interop failure is the right thing to do. That's what NSS does, for instance:
http://searchfox.org/nss/source/lib/ssl/ssl3exthandle.c#2449